### PR TITLE
fix: changed all font family to patternfly specified one

### PR DESCRIPTION
**Pull request by akhilmhdh**
# Fixes

# Explain the feature/fix

1. Changed all font family to patternfly specified font family name

## Does this PR introduce a breaking change

No

## Screenshots

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] My changes generate no new warnings
- [x] I have performed a self-review of my own code
- [x] I have documented my code, particularly in areas like todo, complex logic, quick fix, temporary patch, etc.
#### If it is a new component
- [x] Does your component follow One Platform style guidelines?
- [x] Does your component web accessibility standards? [Helper Doc](https://www.w3.org/TR/wai-aria-1.1/)
- [x] Does your component support desktop screen sizes (350px, 720px, 1150px ,1920px)
#### Browsers you have tested in
- [x] Latest two versions of Mozilla Firefox and Google Chrome supported by Red Hat Enterprise Linux distribution
- [x] Google Chrome [Latest 2 versions]
- [x] Mozilla Firefox [Latest 2 versions]
- [x] Microsoft Edge [Latest 2 versions]
- [x] Apple Safari [Latest 2 versions]
- [x] Microsoft Internet Explorer 11

### CODE REVIEW

**Comment by akhilmhdh** [comment]
@hybridx I haven't bumped some of the packages version because,
1. opc-menu-drawer components will get a release new version in next PR
2. some of there changes where in demo html file only

The one in which gets new version are
1. opc-styles
2. opc-notificaton-drawer
3. opc-feedback
4. opc-back-to-top

File: docs/STYLE-GUIDELINE.md [hunk 1]
```diff
@@ -116,8 +116,8 @@ Use the mixin by `@include <mixin-name>` for example:
 
 ```css
 :root {
```
Comment (hybridx) [inline docs/STYLE-GUIDELINE.md:118]:
```suggestion
:host {
```
I think there would be a change here too

File: packages/opc-back-to-top/src/opc-back-to-top.scss [hunk 2]
```diff
@@ -9,7 +9,7 @@
 }
 
 .show {
-  font-family: "Red Hat Text", sans-serif;
+  font-family: 'RedHatText', sans-serif;
```
Comment (hybridx) [inline packages/opc-back-to-top/src/opc-back-to-top.scss:12]: Is it possible to use the variable here? It's a scss file right?

**Review by ghanlohar — APPROVED** [review]

**Comment by hybridx** [comment]
@akhilmhdh any reason to add all these changes in one pr?

**Review by hybridx — CHANGES_REQUESTED** [review]

File: packages/opc-back-to-top/src/opc-back-to-top.scss (hunk 2 above)
Comment (akhilmhdh) [reply packages/opc-back-to-top/src/opc-back-to-top.scss:12]: Ok will change

**Comment by akhilmhdh** [comment]
@hybridx All the change correspond to using RedHatText and RedHatDisplay

File: docs/STYLE-GUIDELINE.md (hunk 1 above)
Comment (akhilmhdh) [reply docs/STYLE-GUIDELINE.md]: Done

File: packages/opc-back-to-top/src/opc-back-to-top.scss (hunk 2 above)
Comment (akhilmhdh) [reply packages/opc-back-to-top/src/opc-back-to-top.scss]: Done

**Review by gisumit — APPROVED** [review]

**Review by hybridx — APPROVED** [review]

**Pull request merged**